Makefile

```
@@ -217,6 +217,15 @@ e2e-tests: manifests $(KUSTOMIZE) ## Run E2E tests and destroy environment stack
 e2e-tests-keep-stacks: manifests $(KUSTOMIZE) ## Run E2E tests and keep environment stacks running. To run locally, complete pre-reqs (see docs/how-to-contribute.md) and prepend command with `aws-vault exec sso-agent-sandbox-account-admin --`. E.g. `aws-vault exec sso-agent-sandbox-account-admin -- make e2e-tests-keep-stacks`.
 	KUBEBUILDER_ASSETS="$(ROOT)/bin/$(PLATFORM)/" go test -C test/e2e ./... --tags=e2e -run TestLocalKindSuite -v -timeout 0s -coverprofile cover_e2e_keep_stacks.out -args -keep-stacks=true
 
+.PHONY: e2e-test-specific 
+e2e-test-specific:
+	@if [ -z "$(E2ETESTNAME)" ]; then \
+		echo "The environment variable E2ETESTNAME is not set. Please set E2ETESTNAME to the name of the test you want to run."; \
+		exit 1; \
+	else \
+		go test ./test/e2e/tests/... -count=1 --tags=e2e -v -run $(E2ETESTNAME); \
+	fi
+
```

Can we put this logic in the `e2e-tests` directive instead and make the env var optional? Also, what do you think of using the env var name, `E2E_RUN_REGEX` or `TEST_RUN_REGEX`?

```suggestion
.PHONY: e2e-tests 
e2e-tests:
	@if [ -n "$(E2E_RUN_REGEX)" ]; then \
                go test ./test/e2e/tests/... -count=1 --tags=e2e -v -run="$(E2E_RUN_REGEX)"; \
	else \
		go test ./test/e2e/tests/... -count=1 --tags=e2e -v -run="TestAWSKindSuite"; \
	fi

```